roles/composer/defaults/main.yml

```diff
@@ -8,7 +8,7 @@ manala_composer_install_packages_default:
   - unzip
 
 # Version
-manala_composer_version: ~
+manala_composer_version: stable
```

Comment:
Please, reflect in README.md :)
And add a little bafouille on major channel/version/whatever handling

Reply:
done